src/js-context.ts

```diff
@@ -63,6 +63,14 @@ export default class JSContext {
     return methodNames;
   }
 
+  getClassName(): string | null {
```

Comment:
Not sure if this is needed at all... if it is, I guess we might be missing tests for this? ☺️